addons/account_edi_ubl_cii/tests/test_ubl_cii.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.account_edi.tests.common import AccountEdiTestCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestAccountEdiUblCii(AccountEdiTestCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None, edi_format_ref='account_edi_ubl_cii.ubl_bis3'):
+        super().setUpClass(chart_template_ref=chart_template_ref, edi_format_ref=edi_format_ref)
+
+        cls.uom_units = cls.env.ref('uom.product_uom_unit')
+        cls.uom_dozens = cls.env.ref('uom.product_uom_dozen')
+
+        cls.displace_prdct = cls.env['product.product'].create({
+            'name': 'Displacement',
+            'uom_id': cls.uom_units.id,
+            'standard_price': 90.0,
+            'property_account_income_id': cls.company_data['default_account_revenue'].id,
+            'property_account_expense_id': cls.company_data['default_account_expense'].id,
+            'taxes_id': [(6, 0, cls.company_data['default_tax_sale'].ids)],
+        })
+
+        cls.place_prdct = cls.env['product.product'].create({
+            'name': 'Placement',
+            'uom_id': cls.uom_units.id,
+            'standard_price': 80.0,
+            'property_account_income_id': cls.company_data['default_account_revenue'].id,
+            'property_account_expense_id': cls.company_data['default_account_expense'].id,
+            'taxes_id': [(6, 0, cls.company_data['default_tax_sale'].ids)],
+        })
+
+    def test_import_product(self):
+        invoice = self.create_invoice_from_file('account_edi_ubl_cii', 'test_files', 'test_ubl_cii.pdf')
+        self.assertRecordValues(
+            invoice.line_ids[2],
+            [{'product_id': self.place_prdct.id, 'product_uom_id': self.uom_units.id}]
+        )
+        self.assertRecordValues(
+            invoice.line_ids[3],
+            [{'product_id': self.displace_prdct.id, 'product_uom_id': self.uom_units.id}]
+        )
```

Comment:
No need to call `assertRecordValues` each time, you can use it with `invoice.line_ids` and a list of several dict.